pkg/reconciler/instances/istio/action.go

```diff
@@ -1,37 +1,104 @@
 package istio
 
 import (
-	"fmt"
+	"bytes"
 	"github.com/kyma-incubator/reconciler/pkg/reconciler"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/chart"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/file"
 	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
 	"github.com/pkg/errors"
+	"io"
+	"os"
 	"os/exec"
+	"strings"
 )
 
 const (
-	istioctl1_10_2 = "/bin/istioctl-1.10.2"
+	istioctl1_11_1      = "/bin/istioctl-1.11.1" // change path to the local `istioctl` if debugging locally
```

Comment:
Please add ENV variable to change istioctl path. Something like ISTIO_PATH. If the variable is set it has precedence over default location. This way you can test locally without modifying sources.